mtp_bank_admin/apps/bank_admin/tests/test_views.py

```diff
@@ -35,3 +38,47 @@ def check_login_redirect(self, attempted_url):
 
     def test_requires_login_dashboard(self):
         self.check_login_redirect(reverse('bank_admin:dashboard'))
+
+    def test_requires_download_refund_file(self):
+        self.check_login_redirect(reverse('bank_admin:download_refund_file'))
+
+    @mock.patch('bank_admin.refund.api_client')
+    def test_download_refund_file(self, mock_api_client):
+        self.login()
+
+        conn = mock_api_client.get_connection().bank_admin.transactions
+        conn.get.return_value = REFUND_TRANSACTION
+
+        response = self.client.get(reverse('bank_admin:download_refund_file'))
+
+        self.assertEqual(200, response.status_code)
+        self.assertEqual('text/csv', response['Content-Type'])
+        self.assertEqual(b'111111,22222222,DOE JO,25.68,A1234BC 22/03/66\r\n',
+                         response.content)
+
+    @mock.patch('bank_admin.refund.api_client')
+    def test_download_refund_file_error(self, mock_api_client):
+        self.login()
+
+        conn = mock_api_client.get_connection().bank_admin.transactions
+        conn.get.side_effect = Exception('Problem?')
+
+        response = self.client.get(reverse('bank_admin:download_refund_file'),
+                                   follow=True)
+
+        self.assertContains(response, _('Could not download AccessPay file'),
+                            status_code=200)
```

Comment:
200 feels like the wrong status for this? Seems like 503 is what we should be using?